### DISCUSSION TITLE
Ideas to "Invalid input on Endpoint /feeds/2021-06-30/documents"

### DISCUSSION THREAD

**Discussion by Kinggrass** [abuzuhri/Amazon-SP-API-CSharp#14] [Q&A]
Hi @abuzuhri,

I would like to ask you if you have a guess what could be wrong to what I try to discuss on the amazon repository:

[https://github.com/amzn/selling-partner-api-docs/discussions/1524](https://github.com/amzn/selling-partner-api-docs/discussions/1524)

I am somehow stuck with this "small" problem, any ideas?

Edit:

@abuzuhri
Never mind like in the other thread of us the sandbox expects predefined input. So completely useless to try things out with own input.

**Comment by abuzuhri** [comment]
HI @Kinggrass ,

I think the main problem in sandbox try production and see if it work
